### ISSUE TITLE
aws vpc flow-logs Action Tag

### ISSUE THREAD

**Issue by scoleri** [cloud-custodian/cloud-custodian#6978]
### Describe the feature

AWS VPC Flow-logs are taggable.  I don't think currently there is a way to tag this resource with Custodian as the resource is VPC.  

I'd like to be able to tag the flow-logs as an action.  Maybe flow-logs should be a new resource?

**Comment by kapilt** [comment]
agreed re new resource on flow logs, we should also enhance set action flow logs on extant resources with new fields.
